pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+	<!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+	<title>{{title}}</title>
```

Comment:
yeah I know, but I just read in the django tutorial it might be better to use a block in case you want specification for the child that inherits from the base.html file, so we will have the possibility of overriding it, although we for now won't necessarily use it, and it appeared this way in the basic base.html example, you can choose not to override it of course and it will appear the same, that's all

Reply:
I added the block.